File: iOS/watf-bank/PageController/TransferResult.m
#import "TransferResult.h"

@interface TransferResult()
@end

@implementation TransferResult
- (void)viewDidLoad
{
    [super viewDidLoad];
    if(_labelV!=nil)
    {
        _label.text = _labelV;
    }
    _fromAccount.text = _fromAccountV;
    _toAccount.text = _toAccountV;
    _amount.text = _amountV;
}
@end

File: iOS-Swift/watf-bank/PageController/TransferResult.swift
import UIKit

class TransferResult : UIViewController {
    
    @IBOutlet weak var label: UILabel!
    @IBOutlet weak var fromAccount: UILabel!
    @IBOutlet weak var toAccount: UILabel!
    @IBOutlet weak var amount: UILabel!
    
    var labelV = ""
    var fromAccountV = ""
    var toAccountV = ""
    var amountV = ""
    
    override func viewDidLoad() {
        super.viewDidLoad()
        label.text = labelV
        fromAccount.text = fromAccountV
        toAccount.text = toAccountV
        amount.text = amountV
    }
}
